### PR TITLE
test(client): misc tidying

### PR DESCRIPTION
**Pull request by jason-ha**
- routerlicious-urlresolver - remove dup lint settings
- fluid-static - remove extra mocha args
- container-runtime - correct coverage config

### CODE REVIEW

File: packages/drivers/routerlicious-urlResolver/.eslintrc.cjs [hunk 1]
```diff
@@ -23,18 +23,5 @@ module.exports = {
...
-		// This library is used in the browser, so we don't want dependencies on most node libraries.
-		"import/no-nodejs-modules": ["error"],
 	},
-	overrides: [
-		{
-			// Rules only for test files
-			files: ["*.spec.ts", "src/test/**"],
-			rules: {
-				// Test files are run in node only so additional node libraries can be used.
-				"import/no-nodejs-modules": ["error", { allow: ["assert"] }],
```
Comment (ChumpChief) [inline packages/drivers/routerlicious-urlResolver/.eslintrc.cjs:36]: This seems slightly stricter than the one in minimal-deprecated (only allowing assert) - would we want to strengthen the shared config to only allow "assert" too?

File: packages/framework/fluid-static/package.json [hunk 2]
```diff
@@ -65,8 +65,8 @@
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
 		"test:mocha": "npm run test:mocha:esm && echo skipping cjs to avoid overhead - npm run test:mocha:cjs",
-		"test:mocha:cjs": "mocha --recursive \"dist/test/**/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
-		"test:mocha:esm": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
+		"test:mocha:cjs": "mocha --recursive \"dist/test/**/*.spec.*js\" --exit",
```
Comment (ChumpChief) [inline packages/framework/fluid-static/package.json:68]: Most of our packages use mocha-test-setup this way, though I'm always fuzzy on what it's doing - what's the effect of removing it here (and would we do the same for other packages)?

File: packages/runtime/container-runtime/package.json [hunk 3]
```diff
@@ -152,12 +152,12 @@
 		"cache-dir": "nyc/.cache",
 		"exclude": [
 			"src/test/**/*.*ts",
-			"dist/test/**/*.*js"
+			"lib/test/**/*.*js"
```
Comment (ChumpChief) [inline packages/runtime/container-runtime/package.json:155]: It looks like many packages have this same config (using dist) - do they all need to change too?

File: packages/framework/fluid-static/package.json (hunk 2 above)
Comment (DLehenbauer) [reply packages/framework/fluid-static/package.json:68]: Most projects use a '.mocharc.cjs' that already imports the mocha-test-setup:
```ts
const getFluidTestMochaConfig = require("@fluid-internal/mocha-test-setup/mocharc-common");
```

File: packages/drivers/routerlicious-urlResolver/.eslintrc.cjs (hunk 1 above)
Comment (DLehenbauer) [reply packages/drivers/routerlicious-urlResolver/.eslintrc.cjs:36]: For test files, I think general node usage is okay.

Comment (jason-ha) [reply packages/drivers/routerlicious-urlResolver/.eslintrc.cjs:36]: The default is a lot more forgiving. It says any Node use is okay for tests which is intentional and fine. I didn't see value in having this stricter override.

File: packages/runtime/container-runtime/package.json (hunk 3 above)
Comment (jason-ha) [reply packages/runtime/container-runtime/package.json:155]: I think it depends on if they are testing ESM or CJS or both.
I wasn't trying to fix all of the things. Just getting what I saw trying to find good reference when adding tests to core-interfaces.

There are 9 more files with `-r node_modules/@fluid-internal/mocha-test-setup`.
And more that have "dist/test/..." than "lib/test/...".
@DLehenbauer, do you know if there was a debt item to go and tidy?

File: packages/drivers/routerlicious-urlResolver/.eslintrc.cjs (hunk 1 above)
Comment (tylerbutler) [reply packages/drivers/routerlicious-urlResolver/.eslintrc.cjs:36]: This assumes we run all our tests in node, which is not true or it shouldn't be, at least.

Regardless, any change should be in the shared config anyway.

**Pull request closed without merging**